intyg-proxy-service/build.gradle

```diff
@@ -11,6 +11,7 @@ ext {
     jaxsWsVersion = "4.0.1"
     jakartaWsVersion = "4.0.0"
     springbootMavenBomVersion = '3.1.2'
+    guavaVersion = '30.1.1-jre'
```

Comment:
Can you see if we can go for an updated guavaVersion? This version is 2 years old and contains vulnerabilities
https://mvnrepository.com/artifact/com.google.guava/guava/30.1.1-jre

Reply:
Bumped to 32.1.2-jre. Was commonly used and has no vulnerabilities.

https://mvnrepository.com/artifact/com.google.guava/guava